src/TeacherRoster/TeacherRoster.js

```diff
@@ -1,13 +1,64 @@
-import React from 'react'
+import React, { useState } from 'react'
+import Modal from '../Modal/Modal'
+import CreateStudentForm from '../CreateStudentForm/CreateStudentForm'
+import StudentDetails from '../StudentDetails/StudentDetails'
+import { connect } from "react-redux";
 import './TeacherRoster.scss'
 
-const TeacherRoster = () => {
+const TeacherRoster = ({ students }) => {
+  const [ isAddingStudent, toggleAddStudent ] = useState(false)
+  const [ isViewingStudentDetails, toggleStudentDetails] = useState(false)
+  const [ foundStudent, setFoundStudent ] = useState({})
+
+  const renderStudentNames = () => {
+    return students.map(student => {
+      return (
+        <p id={student.id} onClick={(e) => findStudent(e)}>{student.first_name} {student.last_name}</p>
+      )
+    })
+  }
+
+  const findStudent = (e) => {
+    const found = students.find(student => +e.target.id === student.id)
```

Comment:
Whats this +e doing? I have never seen that before

Reply:
Great Catch Bill - I haven't seen that either

Reply:
Making it a number so we don't have to use the disable lint line